controllers/userController.js

```diff
@@ -129,5 +129,24 @@ const userAddress = async (req, res) => {
     res.status(500).send({ message: err })
   }
 }
-
-module.exports = { userRegister, login, getUserData, deleteUserData, limitUsersData, userAddress }
+const deleteUserAddresses = async (req, res) => {
+  const { userAddresses, user_id } = req.body
+  try {
+    if (userAddresses.length === 0) {
+      res.status(404).send({ message: "please provide addressIds to delete the addresses" })
+    } else {
+      const findAllUserAddresses = await addressModel.findAll({ where: { user_id } })
```

Comment:
why do you need to get all user address? when api having address's ids in payload? @ashok-choudhary-0